sonar-php-plugin/src/test/java/org/sonar/plugins/php/PHPSensorTest.java

```diff
@@ -100,7 +101,7 @@ public class PHPSensorTest {
   @org.junit.Rule
   public LogTester logTester = new LogTester();
 
-  private final SensorContextTester context = SensorContextTester.create(new File("src/test/resources").getAbsoluteFile());
+  private SensorContextTester context = SensorContextTester.create(new File("src/test/resources").getAbsoluteFile());
```

Comment:
Just noticed this: Actually, there is no need to initialize this here, as we are calling `resetContext()` in the `before()` method anyway which resets this.